docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -18,8 +18,6 @@ can consume huge pages and the current limitations.
 1. Kubernetes nodes must pre-allocate huge pages in order for the node to report
    its huge page capacity. A node may only pre-allocate huge pages for a single
    size.
-1. A special **alpha** feature gate `HugePages` has to be set to true across the
```

Comment:
L10 says `Kubernetes supports the allocation and consumption of pre-allocated huge pages
 by applications in a Pod as an **alpha** feature.` It should be updated as well.

Reply:
Thanks for catching this. Fixed.